main.js

```diff
@@ -0,0 +1,76 @@
+var snakewaterMontana = {
+    paleontologist: "Dr. Alan Grant",
+    depth: "10 meters",
+    specimen: "Velociraptor"
+  }
+
+  var guestOfHonor = snakewaterMontana.paleontologist;
+  var cleverGirl = snakewaterMontana.specimen; 
+
+  console.log (guestOfHonor);
+  console.log (cleverGirl);
+
+  nicaragua = {
+    depth: "200 meters",
+    annualBudget: 1500000,
+    specimens: [
+      "Tyrannosaurus Rex",
+      "Stegosaurous",
+      "Triceratops",
+      "Velociraptor"
+    ]
+  }
+annualBudget = annualBudget + 250000;
+console.log (annualBudget);
+  var specimens = ['Tyrannosaurus Rex', 'Stegosaurous','Triceratops', 'Velociraptor']
```

Comment:
this is not the way to do it can you imagine if we have a complex array with 100 item and you have to write them manually ? the way to do this is `var nicaraguanSpecimens = nicaragua.specimens;`